### PR TITLE
Robert champion code review

### PR DESCRIPTION
**Pull request by Champ32**
[https://github.com/Odogwufu/TimeMan2/commits/Champ32_Branch]
Program was available on time
Program Compiles
I learned how to setup a contactDTO using Serialized Names

import com.google.gson.annotations.SerializedName

class ContactDTO (
    @SerializedName("firstName") val firstName : String = "",
    @SerializedName("lastName") val lastName  : String = "",
    @SerializedName("email") val email     : String = "",
    @SerializedName("phone") val phone     : String = "", // this value cannot be an Integer due to the size of "Int"
    @SerializedName("favorite") var favorite : Boolean = false)

### CODE REVIEW

File: app/src/main/java/com/plan2meet/dto/GroupDTO.kt [hunk 1]
```diff
@@ -2,6 +2,7 @@ package com.plan2meet.dto
 
 class GroupDTO (
     var calendarDTO    : CalendarDTO?,
+    //make sure to use your variables in your code to lessen technical debt.
```
Comment (discospiff) [inline app/src/main/java/com/plan2meet/dto/GroupDTO.kt:5]: Big no-no: do not put conversation in source code comments.  That's clutter.  Put the conversation here, in the Files Changed section of the pull request, where you can have a dialog with other developers.

File: app/src/main/java/com/plan2meet/ui/RetrofitClientInstance.kt [hunk 2]
```diff
@@ -6,15 +6,16 @@ import retrofit2.converter.gson.GsonConverterFactory
 
 object RetrofitClientInstance {
     private var retrofit: Retrofit? = null
-    private val BASE_URL = ""
+    //never start a variable with an uppercase.
```
Comment (discospiff) [inline app/src/main/java/com/plan2meet/ui/RetrofitClientInstance.kt:9]: 0) The original implementation is correct.  All upper case, separated by underscores, is proper convention for constants.  This appears to be a constant, though I'd add "const" modifier to make that more clear.  
1) Big no-no: do not put conversation in source code comments.  That's clutter.  Put the conversation here, in the Files Changed section of the pull request, where you can have a dialog with other developers.

File: app/src/main/java/com/plan2meet/dto/ContactDTO.kt [hunk 3]
```diff
@@ -8,9 +8,13 @@ class ContactDTO (
     @SerializedName("email") val email     : String = "",
     @SerializedName("phone") val phone     : String = "", // this value cannot be an Integer due to the size of "Int"
     @SerializedName("favorite") var favorite : Boolean = false)
+
+
 {
     @Override
     override fun toString(): String {
-        return "My name is $firstName $lastName, my e-mail is: $email and my phone # is: $phone"
+        //use variable to help clean up return code
```
Comment (discospiff) [inline app/src/main/java/com/plan2meet/dto/ContactDTO.kt:16]: Big no-no: do not put conversation in source code comments.  That's clutter.  Put the conversation here, in the Files Changed section of the pull request, where you can have a dialog with other developers.

File: app/src/main/java/com/plan2meet/dto/ContactDTO.kt [hunk 4]
```diff
@@ -8,9 +8,13 @@ class ContactDTO (
...
+        val contact = "My name is $firstName $lastName, my e-mail is: $email and my phone # is: $phone"
+        return contact
```
Comment (discospiff) [inline app/src/main/java/com/plan2meet/dto/ContactDTO.kt:18]: I'm neutral on this.  Introducing a variable for something used only once does not decrease technical debt.

**Comment by discospiff** [comment]
There's not significant technical debt reduction in this branch, and there is conversation introduced in the source code, so I don't recommend merging this branch.

**Comment by WillTaylor2352** [comment]
As of right now, we will be proceeding with development without using the changes proposed in this pull request

**Pull request closed without merging**